apps/experiments/models.py

```diff
@@ -588,6 +588,7 @@ class Experiment(BaseTeamModel, VersionsMixin):
     )
     debug_mode_enabled = models.BooleanField(default=False)
     citations_enabled = models.BooleanField(default=True)
+    create_version_task_id = models.CharField(max_length=128, blank=True)
```

Comment:
This field's value is not really being used directly, but I think we should keep it like this. Might come in handy if we're wanting to debug something